ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,98 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+  intermediate411:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.11.0-0
+        upper: 4.12.0-0
```

Comment:
Apologize for misleading you:) If we want to upgrade to an officially supported release, then your previous configuration looks right to me.

```
intermediate411:
    release:
      architecture: amd64
      channel: stable
      version: "4.11"
```